src/components/KeepReading/StyledKeepReading.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+export const StyledKeepReading = styled.div`
+  display: flex;
+  border-top: 2px dotted gray;
```

Comment:
Try to stick to the pre-defined colors, and add to the `colors` object if you need to. This will prevent us from having to manually update all this CSS if we decide to alter the color scheme.

```suggestion
  border-top: 2px dotted ${colors.lightGray};
```